discover/deployment/articles/00-deploying-liferay/05-installing-liferay-portal-on-wildfly.markdown

```diff
@@ -0,0 +1,467 @@
+[@product@ Clustering](/discover/deployment/-/knowledge_base/7-1/liferay-clustering)
+for information on setting up a @product@ cluster.
+
+$$$
+
+## Configuring Wildfly [](id=configuring-wildfly)
+
+Configuring Wildfly to run @product@ includes these things:
+
+- Setting environment variables
+- Setting properties and descriptors
+- Removing unnecessary configurations
+
+Optionally, you can configure Wildfly to manage these things for @product@:
+
+- [Data source](#database-configuration)
+- [Mail session](#mail-configuration)
+
+Start with configuring Wildfly to run @product@.
+
+Make the following modifications to
+`$WILDFLY_HOME/standalone/configuration/standalone.xml`:
+
+1.  Locate the closing `</extensions>` tag. Directly beneath that tag, insert
+    the following system properties:
+
+        <system-properties>
+            <property name="org.apache.catalina.connector.URI_ENCODING" value="UTF-8" />
+            <property name="org.apache.catalina.connector.USE_BODY_ENCODING_FOR_QUERY_STRING" value="true" />
+        </system-properties>
+
+2.  Add the following `<filter-spec>` tag within the `<console-handler>` tag,
+    directly below the `<level name="INFO"/>` tag:
+
+        <filter-spec value="not(any(match(&quot;WFLYSRV0059&quot;),match(&quot;WFLYEE0007&quot;)))" />
+
+3.  Add a timeout for the deployment scanner by setting
+    `deployment-timeout="360"` as seen in the excerpt below.
+
+        <subsystem xmlns="urn:jboss:domain:deployment-scanner:2.0">
+            <deployment-scanner deployment-timeout="360" path="deployments" relative-to="jboss.server.base.dir" scan-interval="5000" runtime-failure-causes-rollback="${jboss.deployment.scanner.rollback.on.failure:false}"/>
+        </subsystem>
+
+4.  Add the following JAAS security domain to the security subsystem
+    `<security-domains>` defined in element `<subsystem
+    xmlns="urn:jboss:domain:security:2.0">`.
+
+        <security-domain name="PortalRealm">
+            <authentication>
+                <login-module code="com.liferay.portal.security.jaas.PortalLoginModule" flag="required" />
+            </authentication>
+        </security-domain>
+
+5.  Remove the following Weld-related tags:
+
+    - `<extension module="org.jboss.as.weld"/>`
+    - `<subsystem xmlns="urn:jboss:domain:weld:4.0"/>`
+
+6.  Remove the two code snippets providing welcome content:
+
+        <location name="/" handler="welcome-content"/>
+
+    and
+
+        <handlers>
+            <file name="welcome-content" path="${jboss.home.dir}/welcome-content"/>
+        </handlers>
+
+7.  Find the `<jsp-config/>` tag and set the `development`, `source-vm`, and
+    `target-vm` attributes in the tag. Once finished, the tag should look like
+    this:
+
+        <jsp-config development="true" source-vm="1.8" target-vm="1.8" />
+
+**Checkpoint:**
+
+Before continuing, verify the following properties have been set in the
+`standalone.xml` file:
+
+1.  The new `<system-property>` is created.
```

Comment:
@codyhoag In line 198 we are adding `<system-properties>` element with `<property>` child elements; I think it's better to say "The new `<system-properties>` element is added` to be consistent.

I may miss this part in the JBoss doc.

cc @alee8888 Other things look good to me.

Reply:
@dantewang makes sense; I'll update it. Thanks!